Firmware/Marlin_main.cpp

```diff
@@ -3166,10 +3200,7 @@ void gcode_M701()
 		plan_buffer_line(current_position[X_AXIS], current_position[Y_AXIS], current_position[Z_AXIS], current_position[E_AXIS], 400 / 60, active_extruder); //fast sequence
 		st_synchronize();
 
-		marlin_rise_z();
-		current_position[E_AXIS] += 30;
```

Comment:
Why didn't you keep this line? With these changes my filament only loads to the nozzle and doesn't extrude any filament. I have to click NO and only then does it start extruding